examples/webserver-comp/webserver.ts

```diff
@@ -29,14 +28,23 @@ let group = new aws.ec2.SecurityGroup("web-secgrp-comp", {
     ],
 }, providerOpts);
 
+const ami = aws.ec2.getAmiOutput({
+    owners: ["amazon"],
+    mostRecent: true,
+    filters: [{
+        name: "name",
+        values: ["amzn2-ami-hvm-*-x86_64-ebs"],
```

Comment:
Should we use the latest al2023 AMIs instead? I think it is `al2023-ami-kernel-default-x86_64`

Reply:
Yeah that's a good suggestion! They're supported even longer (2025 vs 2028)

Reply:
Done! :)